packages/api/src/schema/ai-api-schema.yaml

```diff
@@ -368,6 +368,65 @@ paths:
               schema:
                 $ref: '#/components/schemas/studio-api-error'
       x-speakeasy-name-override: segmentAnything2
+  /api/generate/llm:
```

Comment:
Keep in mind you might have conflict with @mjh1's #2319, but it should be solved by simply running the `pull-ai-schema` script again